SpatialGDK/Source/SpatialGDKEditor/Private/WorkerTypeCustomization.cpp

```diff
@@ -0,0 +1,83 @@
+#include "WorkerTypeCustomization.h"
+
+#include "SpatialGDK/public/SpatialGDKSettings.h"
+
+#include "PropertyEditor/Public/DetailLayoutBuilder.h"
+#include "PropertyEditor/Public/DetailWidgetRow.h"
+#include "PropertyEditor/Public/IDetailChildrenBuilder.h"
```

Comment:
Can you make this #include "IDetailChildrenBuilder.h"? I don't think the full path has to be there.